src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.BatchAI</PackageId>
 		<Description>Provides management capabilities for Azure Batch AI.</Description>
-		<Version>2.0.0</Version>
+		<Version>2.0.1</Version>
```

Comment:
Since the enum values are being renamed here, it is a breaking change. Please update the major version number here